src/sampling.jl

```diff
@@ -68,7 +68,8 @@ end
 #
 #   Reference: The Art of Computer Programming, Vol 2, 3.4.2, p.142
 #
-function knuths_sample!(a::AbstractArray, x::AbstractArray; initshuffle::Bool=true)
+function knuths_sample!(a::AbstractArray, x::AbstractArray, rng::AbstractRNG=Base.GLOBAL_RNG;
+                       initshuffle::Bool=true)
```

Comment:
It would be great if you could indent the lines in the function signatures such that they align after the initial `(`. That is, this would look like
```julia
function knuths_sample!(a::AbstractArray, x::AbstractArray, rng::AbstractRNG=Base.GLOBAL_RNG;
                        initshuffle::Bool=true)
```